Leanplum/Manager/LPRequestCallbackManager.h

```diff
@@ -0,0 +1,28 @@
+//
+//  LPRequestCallbackManager.h
+//  Leanplum
+//
+//  Created by Grace on 6/18/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LPRequestCallbackManager : NSObject
+
+@property (nonatomic, strong) NSMutableDictionary *successBlocks;
+@property (nonatomic, strong) NSMutableDictionary *failureBlocks;
```

Comment:
agree to Mayank, these needs to be copy for both success and failure and be made private variables.